.github/dependabot.yaml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
maybe weekly would put less toll on the maintainers?

Reply:
That sounds fair enough, it could always be increased in the future.